### PR TITLE
chore: add susds streams

### PR DESCRIPTION
**Pull request by gosuto-inzasheru**
also have draft csvs for aurabal and sdbal

Screenshot 2024-10-09 at 18 46 29

### CODE REVIEW

File: MaxiOps/Payroll_Txns/Q4_2024/susds_streams_24q4-25q1.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+address,amount,start,end
+0x99AfD53f807766A8B98400B0C785E500c041F32B,77702.394,2024-10-01 02:00,2025-03-31 01:59
+0x27e472f2625D6d4913F6cb5b99DAAadb58Da1D93,80690.947,2024-10-01 02:00,2025-03-31 01:59
```
Comment (Tritium-VLK) [inline MaxiOps/Payroll_Txns/Q4_2024/susds_streams_24q4-25q1.csv:3]:
```suggestion
```

**Review by Tritium-VLK — CHANGES_REQUESTED** [review]
Count me out.  This isn't well enough thought out.

File: MaxiOps/Payroll_Txns/Q4_2024/susds_streams_24q4-25q1.csv (hunk 1 above)
Comment (Tritium-VLK) [reply MaxiOps/Payroll_Txns/Q4_2024/susds_streams_24q4-25q1.csv:3]: This is unpopular with my colleagues and is feared to create too much work,.  Nevermind.  I can rock with the pack.

**Review by Tritium-VLK — APPROVED** [review]

**Comment by gosuto-inzasheru** [comment]
Screenshot 2024-10-10 at 16 46 59

**Pull request merged**